Code/GraphMol/MolDraw2D/MolDraw2DUtils.cpp

```diff
@@ -438,7 +438,18 @@ void drawMolACS1996(MolDraw2D &drawer, const ROMol &mol,
         << " and that may not look great with a pre-determined size."
         << std::endl;
   }
-  double meanBondLen = MolDraw2DUtils::meanBondLength(mol, confId);
+  double meanBondLen = 1.0;
+  if (drawer.drawOptions().prepareMolsBeforeDrawing &&
+      not mol.getNumConformers()) {
```

Comment:
`not` isn't standard C++. This was causing windows build failures
```suggestion
       !mol.getNumConformers()) {
```